tests/test_operators.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import numpy as np
+from ai4pdes.operators import get_weights_linear_2D
+
+# x-advection stencil/ filter with dx=0.5
+ADV_FILTER = np.array([[
+    [[ 0.1667],
+    [ 0.0000],
+    [-0.1667]],
+
+    [[ 0.6667],
+    [ 0.0000],
+    [-0.6667]],
+
+    [[ 0.1667],
+    [ 0.0000],
+    [-0.1667]]
+]])
```

Comment:
generally wouldn't hardcode this, go for high level properties instead, e.g.

- sum(row) = 0, sign swap left right, sum(ADV_FILTER) should be (close to) 0, i.e. < eps(float32)